site.js

```diff
@@ -1239,24 +1321,44 @@ function generateCurse() {
 		},
 		{
 			subjectText: "hyena",
-			makeAdditionalExplaination: function(){return decidedAndTrue(subjectFemale) 
-				? "You have a pseudopenis, complete with a fatty sack standing in for testicles." 
-				: "Remember: male hyenas are submissive to the females.";},
+			chosen: function(){facialfeatureName = randomFrom(["grinning muzzle", "ears"]);},

```

Comment:
Maybe bone-crushing jaws or muscular jaws? You can also modify ears with "paddle-like" or something. : )